src/PaymentMethodCollectElement.res

```diff
@@ -311,14 +340,20 @@ let make = (~integrateError, ~logger) => {
           {React.string(statusInfo.message)}
         </div>
         <div className="flex border-t border-bg-jp-gray-300 py-[20px] w-full justify-center">
-          <div className="flex flex-col max-w-[500px] bg-white w-full mx-[40px]">
+          <div
+            className="flex flex-col max-w-[500px] bg-white w-full mx-[10px]
+            xs:mx-[40px]">
             {statusInfoFields
             ->Array.mapWithIndex((info, i) => {
-              <div key={i->Int.toString} className={`flex flex-row items-center`}>
-                <div className="text-[15px] text-jp-gray-900 min-w-[10ch] text-right">
+              <div key={i->Int.toString} className={`flex flex-row items-center mb-[4px]`}>
+                <div
+                  className="text-[14px] text-jp-gray-900 text-right min-w-[12ch]
```

Comment:
Could you please use rem instead of ch. Also could you please use theme to get the fontSize to make the appearance customizable